documentation/TechnicalDesignDocument.tex

```diff
@@ -44,7 +44,25 @@
 
     %------------------------------------------------------------------------------------------------------------------------------------------------------- Requirements
     \section{Requirements}
-    Functional and Non-functional requirements.
+    In this section the various functional and non-functional requirements the program has are listed.
+    \subsection{Functional Requirements:}
+    \begin{itemize}
+        \item The program should support loading different maramafications using the .mar files the marama-transpiler creates.
```

Comment:
```
The program should support loading different maramafications using the .mar files the marama-transpiler creates.
```

This item is something specific to the Editor. Could you maybe optimize the list so it only talks about requirements related to the View?